platforms/posix/CMakeLists.txt

```diff
@@ -62,10 +62,11 @@ else()
 			${df_driver_libs}
 			df_driver_framework
 			pthread m
-			
+
 			# horrible circular dependencies that need to be teased apart
-			px4_layer
-			px4_platform
+			px4_layer px4_platform
```

Comment:
Do you want to insert a newline here instead of a space?

Reply:
No, they're both part of the horrible circular dependency mess that needs to be fixed.